chia/_tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -405,29 +412,25 @@ async def test_push_transactions(wallet_rpc_environment: WalletRpcTestEnvironmen
         )
     ).signed_tx
 
-    resp_client = await client.push_transactions([tx], fee=uint64(10))
+    resp_client = await client.push_transactions(
+        PushTransactions(transactions=[tx], fee=uint64(10)),  # pylint: disable=unexpected-keyword-arg
+        DEFAULT_TX_CONFIG,
+    )
     resp = await client.fetch(
         "push_transactions", {"transactions": [tx.to_json_dict_convenience(wallet_node.config)], "fee": 10}
     )
     assert resp["success"]
-    resp = await client.fetch("push_transactions", {"transactions": [tx.to_json_dict()], "fee": 10})
+    resp = await client.fetch("push_transactions", {"transactions": [bytes(tx).hex()], "fee": 10})
```

Comment:
are we changing the api?  or did `tx.to_json_dict()` create a string before?

Reply:
Actually, this is a test coverage change.  The client used to send bytes across and the two lines below the first added coverage to when it sends the two types of dicts across.  I changed the client to start sending dicts across so now I changed the line that was adding coverage for the dicts to instead add coverage for the bytes.

Reply:
So the `transactions` field already was a union effectively and remains unchanged in what is properly handled there?

Reply:
Correct